PsfLauncher/main.cpp

```diff
@@ -74,16 +65,17 @@ int launcher_main(PCWSTR args, int cmdShow) noexcept try
 
     // At least for now, configured launch paths are relative to the package root
     std::filesystem::path packageRoot = PSFQueryPackageRootPath();
+	auto currentDirectory = (packageRoot / dirStr);
+
+	PsfPowershellScriptRunner powershellScriptRunner(appConfig, currentDirectory);

```

Comment:
>PsfPowershellScriptRunner [](start = 1, length = 25)

nit: newlines

Reply:
Fixed.

---
In reply to: [316806377](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/79#discussion_r316806377) [](ancestors = 316806377)